### ISSUE TITLE
[Navigation Drawer] Update the semi static menus to use the configuration form the config_navigation_drawer.json

### ISSUE THREAD

**Issue by dubdabasoduba** [opensrp/fhircore#1371]
**Description of feature**
- After updating the [navigation drawer UI](https://github.com/opensrp/fhircore/issues/1366), we would like to make sure the menus on the Navigation drawer adhere to the rules defined on the `config_navigation_drawer.json`

**Describe the enhancement**
- [x] Update the following menu functionality to adhere to the rules defined on the `staticMenu` attributes on the [`config_navigation_drawer.json`](https://github.com/opensrp/fhir-resources/blob/main/sample_project/app_configurations/config_navigation_drawer.json)
    - [x]  [Main Add button](https://github.com/opensrp/fhir-resources/blob/f9b595f8170f7686746766f5ca2b8c67bd36f6b6/sample_project/app_configurations/config_navigation_drawer.json#L5-L23). Please ignore the `action` attribute functionality here. This will be handled in another issue. 
    - [x] [Reports](https://github.com/opensrp/fhir-resources/blob/f9b595f8170f7686746766f5ca2b8c67bd36f6b6/sample_project/app_configurations/config_navigation_drawer.json#L24-L28)
    - [x] [P2P Sync](https://github.com/opensrp/fhir-resources/blob/f9b595f8170f7686746766f5ca2b8c67bd36f6b6/sample_project/app_configurations/config_navigation_drawer.json#L29-L33)
    - [x] [Profile/Settings](https://github.com/opensrp/fhir-resources/blob/f9b595f8170f7686746766f5ca2b8c67bd36f6b6/sample_project/app_configurations/config_navigation_drawer.json#L34-L38)  
    - [x] [Other Patients/Clients](https://github.com/opensrp/fhir-resources/blob/ebd4cfe6f873288fc4dc4826c395fa8d1fcb31a9/sample_project/app_configurations/config_navigation_drawer.json#L94-L96)

- [x] The rules define the following items on the configuration of the menus.
    - [x] Visible --> Denotes whether the menu is visible of not
    - [x] Display --> This is the name shown to the users
    - [x] Questionnaires --> This is only available for menus that load up questionnaires.  This is meant to replace the [config_forms.json](https://github.com/opensrp/fhir-resources/blob/main/ecbis/app_configurations_debug/config_forms.json) once the configurability implementation is complete. 
    - [x] Other Patient --> `visible` & `display` attribute follow the above functionality. The `bottomSheetRegister` describes the registers that are shown on the bottom drawer. In case the `bottomSheetRegister` array is blank then also automatically set the `visible` to `false`
 
[**Navigation Drawer Configuration Discussion**](https://github.com/opensrp/fhircore/discussions/1304)

**Depends On**
- [x] https://github.com/opensrp/fhircore/issues/1366

**Comment by f-odhiambo** [comment]
Marked as DONE
